libs/dh/reports/feature-measurement-reports/src/components/measurement-reports.component.ts

```diff
@@ -16,111 +16,94 @@
 
-type DhMeasurementReport = {
-  id: string;
-  startedAt: Date;
-  actorName: string;
-  meteringPoints: string[];
-  gridAreas: string[];
-  statusType: string;
-};
+import { DhNewReportRequest } from './new-report-request.component';
+import { DhOverview } from './overview/overview.component';
 
 @Component({
   selector: 'dh-measurement-reports',
   imports: [
     TranslocoDirective,
+    WATT_CARD,
 
-    WATT_TABLE,
-    WattDataTableComponent,
+    VaterStackComponent,
+    VaterFlexComponent,
     VaterUtilityDirective,
-    WattDatePipe,
+    VaterSpacerComponent,
+    WattEmptyStateComponent,
+    WattSpinnerComponent,
+    DhNewReportRequest,
+    DhOverview,
   ],
-  template: `
-    <watt-data-table
-      *transloco="let t; read: 'reports.measurementReports'"
-      vater
-      inset="ml"
-      [enableSearch]="false"
-    >
-      <h3>{{ t('title') }}</h3>
-
-      <watt-table
-        [dataSource]="dataSource"
-        [columns]="columns"
-        [displayedColumns]="displayedColumns()"
-      >
-        <ng-container
-          *wattTableCell="columns['startedAt']; header: t('columns.startedAt'); let entry"
-        >
-          {{ entry.startedAt | wattDate: 'long' }}
-        </ng-container>
+  styles: `
+    :host {
+      display: block;
+    }
 
-        <ng-container
-          *wattTableCell="columns['actorName']; header: t('columns.actorName'); let entry"
-        >
-          {{ entry.actorName }}
-        </ng-container>
+    h3 {
+      margin: 0;
+    }
+  `,
+  template: `
+    <watt-card vater inset="ml" *transloco="let t; read: 'reports.measurementReports'">
+      @if (isLoading()) {
+        <vater-stack fill="vertical" justify="center">
+          <watt-spinner />
+        </vater-stack>
+      } @else {
+        @if (totalCount() === 0) {
+          <vater-stack fill="vertical" justify="center" gap="l">
+            <watt-empty-state
+              [icon]="hasError() ? 'custom-power' : 'custom-no-results'"
+              [title]="hasError() ? t('errorTitle') : ''"
+              [message]="hasError() ? t('errorMessage') : t('emptyMessage')"
+            >
+              @if (hasError() === false) {
+                <dh-new-report-request />
+              }
+            </watt-empty-state>
+          </vater-stack>
+        } @else {
+          <vater-flex fill="vertical" gap="ml">
+            <vater-stack direction="row" gap="s">
+              <h3>{{ t('title') }}</h3>
+              <span class="watt-chip-label">{{ totalCount() }}</span>
 
-        <ng-container
-          *wattTableCell="columns['meteringPoints']; header: t('columns.meteringPoints'); let entry"
-        >
-          --
-        </ng-container>
+              <vater-spacer />
 
-        <ng-container
-          *wattTableCell="columns['gridAreas']; header: t('columns.gridAreas'); let entry"
-        >
-          @let gridAreas = entry.gridAreas;
+              <dh-new-report-request />
```

Comment:
Suggestion: Should this not be moved into the data table as an action. Data table has a count as well

Reply:
The PR moves away from Watt data table because Pernille specifically wanted the empty state to look like the one in Settlement reports.

Reply:
Maybe i could extend data table with the empty state -> action button pattern? if that something we are going to use in multiple places

Reply:
I think it'll require a lot of effort to make the `watt-data-table` support a completely clean empty view similar to Settlement reports. That component displays a few things by default that will need to be toggled, which will increase its complexity.